xbmc/filesystem/AndroidAppFile.h

```diff
@@ -56,6 +56,7 @@ class CFileAndroidApp : public IFile
 private:
   CURL              m_url;
   std::string       m_appname;
+  androidPackage    m_droidPackage;
   int               m_iconWidth;
```

Comment:
if you make this a ptr (androidPackage *m_droidPackage) then you can forward declare it and remove the include of android/activity/XBMCApp.h. Sure a little more work inside the cpp but filesystem includes tend to spew all over the place.